### ISSUE TITLE
Flag to exclude node_modules

### ISSUE THREAD

**Issue by techsin** [abou7mied/detect-circular-deps#3]
Maybe a flag to include or exclude node_modules imports/require.

Sometimes some packages in node_module mess up and prevents detect-circular-deps from finishing.

`detect-circular-deps --inc_nm ./**/*.js `

something like that

**Comment by abou7mied** [comment]
Actually **node_modules** are excluded by default and the tool doesn't run on them.
https://github.com/abou7mied/detect-circular-deps/blob/0b2e663b2789622abb2f758814235d2f2f61e11d/cli.js#L25

I just forgot and removed the line which exit the process after the detection completed
https://github.com/abou7mied/detect-circular-deps/blob/650371af3939776d6b22fc45ec239370d8075d80/cli.js#L102

I just published a patch [v1.2.1](https://github.com/abou7mied/detect-circular-deps/releases/tag/v1.2.1), please test it out and inform me if the problem still exists.

**Comment by techsin** [comment]
when I run 

`detect-circular-deps ./**/*.js`

I get this

```

C:\Users\xyz\AppData\Roaming\npm\node_modules\detect-circular-deps\lib\index.js:110
  const actual = completeExports[propName];
                                ^

TypeError: Cannot read property '__esModule' of undefined
    at validateProperty (C:\Users\xyz\AppData\Roaming\npm\no
......
```

node: v9.3.0
detect-circular-deps@1.2.1

**Comment by abou7mied** [comment]
I couldn't reproduce this error using the same node version. Could you write a sample of what you run the tool on please?

Do you use .babelrc config in your project? if you don't do so, I think I should add a flag to enable **babel-register** and make it disabled by default.

**Comment by abou7mied** [comment]
Oh, I just discovered that the **node_modules** directory isn't excluded properly. I tried to debug ```detect-circular-deps './**/*.js'``` and discovered that the **node_modules** is still included.

I will publish a patch very soon.

**Comment by abou7mied** [comment]
I just published a patch [v1.2.2](https://github.com/abou7mied/detect-circular-deps/releases/tag/v1.2.2), please test it out.

**Comment by techsin** [comment]
`$ detect-circular-deps api/**/*.js`

```
C:\Users\techsin\AppData\Roaming\npm\node_modules\detect-circular-deps\lib\index.js:110
  const actual = completeExports[propName];
                                ^

TypeError: Cannot read property '__esModule' of undefined
```

```
$ node -v
v9.3.0


$ npm list -g | grep detect-circular
+-- detect-circular-deps@1.2.2

windows 10
```

**Comment by abou7mied** [comment]
That's very strange, completeExports shouldn't be undefined at all! 
Although I'm using Linux, I tried to run it on windows with the same node version, but it didn't give me such an error.

Anyway I published a patch [v1.2.3](https://github.com/abou7mied/detect-circular-deps/releases/tag/v1.2.3) for that specific issue.

Please check it out and inform me with results.

Thank you for contributing.
